python/tvm/autotvm/measure/measure.py

```diff
@@ -101,20 +101,29 @@ def measure_option(mode,
         The number of measurement task that can run in parallel.
         Set this according to the number of cpu cores (for compilation) and
         the number of devices you have (for measuring generate code).
+    do_fork: bool, optional
```

Comment:
Principle one of interface design is to simplify and hide options user do not use, in this case, do_fork is only used in local. I think we should remove this, and allow user to pass in 

measure_func = autotvm.measure.local_nofork(measure args)

Reply:
similarly, if pack_size, rpc_device_key etc are only arguments to rpc. I think we should have good default, and allow user to do

```python
measure_func = autotvm.measure.rpc_(rpc_key=xxxx)
```

Reply:
`do_fork` is used in local_executor not measure_func. It can be used in any mode.